pubspec.yaml

```diff
@@ -1,12 +1,18 @@
 name: tflite_native
-version: 0.3.0
+version: 0.3.1-dev
```

Comment:
Why -dev?

Reply:
So we know it's after `0.3.0` but not ready to publish yet.

Reply:
Cool, thanks for the changes :)